src/data/modifiers.txt

```diff
@@ -2714,6 +2714,7 @@ Item	astral ring	Mysticality Percent: +25, Spell Damage: +20, MP Regen Min: 4, M
 Item	atlas of local maps	Combat Rate: -5, Lasts Until Rollover
 # attorney's badge: OBJECTION!
 Item	attorney's badge	Single Equip
+Item	automatic wine thief	Single Equip
```

Comment:
Doesn't the description tell you what this does so that you can put a comment here? (Same for the other equipment)